idea_town/frontend/static-src/app/lib/page-manager.js

```diff
@@ -37,6 +41,16 @@ export default class PageManager {
       const Ctor = this.pages[page];
       const newPage = new Ctor(opts);
 
+      if (newPage.headerScroll) {
```

Comment:
@lmorchard I made a patch here to see what the subviews in the base view would look like. https://gist.github.com/meandavejustice/4106d276de3a9262da0d The other method looks interesting as well